pom.xml

```diff
@@ -4,9 +4,38 @@
     <groupId>com.coveros</groupId>
     <artifactId>selenified</artifactId>
     <version>3.0.1-SNAPSHOT</version>
+
+    <prerequisites>
+        <maven>3.0.4</maven>
+    </prerequisites>
+
     <properties>
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
         <suiteXmlFile>integration.xml</suiteXmlFile>
```

Comment:
as long as we're refactoring, one of the things I'd like to do is remove the need for this integration.xml file
We can put the information from this file directly into the failsafe configuration

Reply:
I'll look at it next round as part of trying to self-contain the integration tests in Maven.